src/legacy/core_plugins/vis_default_editor/public/components/agg.tsx

```diff
@@ -45,8 +46,11 @@ export interface DefaultEditorAggProps extends DefaultEditorAggCommonProps {
   isLastBucket: boolean;
   isRemovable: boolean;
   setAggsState: React.Dispatch<AggsAction>;
+  timeRange?: TimeRange;
 }
 
+export const DefaultEditorAggParamsContext = createContext({} as any);
```

Comment:
We're not using `any` when it's avoidable. This context should be properly typed, please :-)